sktime/forecasting/trend.py

```diff
@@ -16,9 +16,103 @@
 from sktime.utils.datetime import _get_duration
 
 
-class PolynomialTrendForecaster(BaseForecaster):
+class TrendForecaster(BaseForecaster):
+    """Forecast time series data.
+
+    Default settings train a linear regression model.
+
+    Parameters
+    ----------
+    regressor : estimator object, optional (default = None)
+        Define the regression model type. If not set, will default to
+         sklearn.linear_model.LinearRegression
+
+    Example
+    ----------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.trend import TrendForecaster
+    >>> y = load_airline()
+    >>> forecaster = TrendForecaster()
+    >>> forecaster.fit(y)
+    TrendForecaster(...)
+    >>> y_pred = forecaster.predict(fh=[1,2,3])
     """
-    Forecast time series data with a polynomial trend.
+
+    _tags = {
+        "univariate-only": True,
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,
+    }
+
+    def __init__(self, regressor=None):
+        self.regressor = regressor
```

Comment:
I think this should be a `clone`, i.e., use `sklearn` `clone` to make a copy to `regressor_`. No need to have two variables, `regressor` and `regressor_`

Reply:
Just to clarify, cloning happens in `_fit`, the `__init__` method just remembers the input.